### PR TITLE
remove the -c and -n flags that control log compaction

### LINKED ISSUE #552: remove -n option as non functional and redundant
There is no point to keep that option.

> -n option was only useful during development of the
compaction feature to compare its performance and disk
use to the older non-compacting behavior.
Which reminds me, we should remove that option now.
There's no reason for anyone to use it.
_Originally posted by @kr in https://github.com/beanstalkd/beanstalkd/issues/15#issuecomment-11368695_

### PR DESCRIPTION
**Pull request by ysmolsky**
Log compaction should not be switched off.

Fixes #552

### CODE REVIEW

File: util.c [hunk 1]
```diff
@@ -109,8 +109,8 @@ usage(int code)
             " -z BYTES set the maximum job size in bytes (default is %d, max allowed is %d)\n"
             " -s BYTES set the size of each write-ahead log file (default is %d)\n"
             "            (will be rounded up to a multiple of 4096 bytes)\n"
-            " -c       compact the binlog (default)\n"
-            " -n       do not compact the binlog\n"
+            " -c       (DEPRECATED) compact the binlog (default)\n"
+            " -n       (DEPRECATED) do not compact the binlog\n"
```
Comment (sergeyklay) [inline util.c]: The "deprecated" term means "still works but not recommended", but the behavior was actually changed, isn't?

Comment (ysmolsky) [reply util.c]: Oh right, bad choice of words. In fact it was completely removed, not only discouraged.

**Comment by ysmolsky** [comment]
I have removed these flags from the help because they should not be used. In case if somebody was using them, he will get a warning.

**Comment by ysmolsky** [comment]
@kr thank you. I have fixed manuals.

File: doc/beanstalkd.ronn [hunk 2]
```diff
@@ -96,6 +83,17 @@ and format of the `beanstalkd` protocol.
 * `-z` <bytes>:
   The maximum size in bytes of a job.
 
+* `-c`:
+  This flag has no effect. It is kept for historical compatibility only.
+
+  Perform online, incremental compaction of binlog files. Negates
```
Comment (kr) [inline doc/beanstalkd.ronn]: Do you think this line should be deleted?

Comment (ysmolsky) [reply doc/beanstalkd.ronn]: They have no meaning. Removed them.

**Pull request merged**